frameworks/AutoWEKA/setup.sh

```diff
@@ -1,18 +1,38 @@
 #!/usr/bin/env bash
 HERE=$(dirname "$0")
 VERSION=${1:-"stable"}
+WEKA=${2:-"https://prdownloads.sourceforge.net/weka/weka-3-8-5-azul-zulu-linux.zip"}
+
 . ${HERE}/../shared/setup.sh "${HERE}"
 if [[ -x "$(command -v apt-get)" ]]; then
     SUDO apt-get update
     SUDO apt-get install -y wget unzip openjdk-8-jdk
 fi
 
-AUTOWEKA_ARCHIVE="autoweka-${VERSION}.zip"
+if [[ "$VERSION" == "latest" || "$VERSION" == "stable" ]]; then
```

Comment:
Shouldn't "stable" be downloaded from `www.cs.ubc.ca`?
```suggestion
if [[ "$VERSION" == "latest" ]]; then
```

Reply:
The official 2.6.3 release is only on Github (for now?). Per [this autoweka issue](https://github.com/automl/autoweka/issues/94#issuecomment-871334814) it's the official latest release. From my understanding there won't be dev releases under this link (there is no history of dev releases and the project is officially non-active).

Reply:
Would you agree that's enough reason to download `stable` from the same source, or do we stick to `www.cs.ubc.ca` (and manually update it with new versions)?

Reply:
if the project is officially inactive, and therefore latest=stable, then it's fine with me, will approve the changes.

Reply:
from the readme: 
 >This project will not be updated anymore. We may make exceptions for pull requests.